content/docs/README.md

```diff
@@ -143,6 +167,86 @@ To add a single page <https://example.com/changelog> to the docs sidebar, add th
 
 All available languages for code blocks can be found [here](https://prismjs.com/index.html#supported-languages).
 
+## Code Tabs
+
+To display code tabs, wrap all pieces of code with `<CodeTabs></CodeTabs>` and write labels of code tabs in order:
+
+````md
+<CodeTabs labels={["Shell", "C++", "C#", "Java"]}>
+
+```bash
+#!/bin/bash
+STR="Hello World!"
+echo $STR
+```
+
+```c++
+#include <iostream>
+
+int main() {
+    std::cout << "Hello World";
+    return 0;
+}
+```
+
+```csharp
+namespace HelloWorld
+{
+    class Hello {
+        static void Main(string[] args)
+        {
+            System.Console.WriteLine("Hello World");
+        }
+    }
+}
+```
+
+```java
+import java.io.*;
+
+class GFG {
+    public static void main (String[] args) {
+       System.out.println("Hello World");
+    }
+}
+```
+
+</CodeTabs>
+````
+
+<details>
+<summary>Examples</summary>
+
+![Code tabs example](code-tabs-example.jpg)
+
+</details>
+
+## Admonition
+
+In addition to emphasize your text, wrap text with `<Admonition></Admonition>` and set the proper type.
+
+There are 5 types of Admonition: `note`, `important`, `tip`, `warning`, `info`.
+
+You may also specify an optional title with prop `title`.
+
+Example:
+
+```md
+<Admonition type="note" title="Your title">
+The branch creation process does not increase load on the originating project. You can create a branch at any time without worrying about downtime or performance degradation.
+</Admonition>
+<Admonition type="info">
+The branch creation process does not increase load on the originating project. You can create a branch at any time without worrying about downtime or performance degradation.
+</Admonition>
+```
```

Comment:
```suggestion
```md
<Admonition type="note" title="Your title">
  The branch creation process does not increase load on the originating project. You can create a branch at any time without worrying about downtime or performance degradation.
</Admonition>

<Admonition type="info">
  The branch creation process does not increase load on the originating project. You can create a branch at any time without worrying about downtime or performance degradation.
</Admonition>
```
```